src/userMessage/userMessage.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import { StoryblokService } from '../botMessage/storyblok.service';
+import { ConversationService } from '../conversation/conversation.service';
+
+@Injectable()
+export class UserMessageService {
+  constructor(
+    private conversationService: ConversationService,
+    private storyblokService: StoryblokService,
+  ) {}
+
+  setupConversation = async (
+    userResponse,
+    conversationId,
+    previousMessageStoryblokId,
+  ): Promise<void> => {
+    // TODO: Can we do something nice with the getBotResponsesBySlug
+    // so we don't have to filter afterwards.
+    const splitUserResponse = userResponse.split('-');
+    const botResponses = await this.storyblokService.getBotResponsesBySlug(
+      'setup',
+    );
+
+    const previousMessageWasSetupMessage =
+      botResponses.filter(
+        response => response['uuid'] === previousMessageStoryblokId,
+      ).length > 0;
```

Comment:
An alternative (achieves the same thing):

```suggestion
    const previousMessageWasSetupMessage =
      botResponses.some(
        response => response['uuid'] === previousMessageStoryblokId,
      );
```